spec/core/computer_player_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "core/players/computer_player"
+require "console/console"
+require "console/console_io"
+require "console/messages"
+
+describe ComputerPlayer do
+  let(:messages) { Messages.new }
+  it "has correct symbol" do
+    player = ComputerPlayer.new(:X)
+    expect(player.symbol).to eq(:X)
+  end
+
+  it "takes top left corner if not already taken" do
+    player = ComputerPlayer.new(:X)
+    board = Board.new
+
+    expect(player.request_move(board)).to eq(1)
+  end
+
+  it "request move via minimax" do
```

Comment:
If this method tests that the move is requested via minimax, how are you asserting that? When I read the test name I expected that you would use a spy to assert that.
Maybe the test name should just be "computer makes a move" or something like that?